app/src/main/java/com/infomaniak/mail/ui/main/folder/ThreadListFragment.kt

```diff
@@ -529,10 +530,14 @@ class ThreadListFragment : TwoPaneFragment(), SwipeRefreshLayout.OnRefreshListen
     }
 
     private fun observeNetworkStatus() {
-        mainViewModel.isInternetAvailable.observe(viewLifecycleOwner) { isAvailable ->
-            TransitionManager.beginDelayedTransition(binding.root)
-            binding.noNetwork.isGone = isAvailable
-            if (!isAvailable) updateThreadsVisibility()
+        lifecycleScope.launch {
```

Comment:
Can you please specify the Dispatcher ?

Reply:
The `Dispatchers` is not important cause we're already in the main Dispatchers, but it's the wrong scope, use `viewLifecycleOwner.lifecycleScope` instead